components/ImagesGallery/ImagesGallery.vue

```diff
@@ -602,7 +586,7 @@ export default {
     findEntryWithPathInArray(array, path) {
       if (path && array) {
         for (let i = 0; i < array.length; i++) {
-          if (path === array[i].dataset.path) return array[i]
+          if (path === array[i].dataset.path.replace(/ /g, '_')) return array[i]
```

Comment:
why is the .replace necessary here and below? Does this have something to do with name mangling that we were seeing before where certain characters were getting replaced with underscores?

Reply:
Yes it is to do with the name mangling.  I think it is best if I take it out of here and deal with it elsewhere.

Reply:
Is it something that can be resolved by adding file names to this list: https://github.com/nih-sparc/sparc-api/blob/master/app/manifest_name_to_discover_name.py